### PR TITLE
Dev sign in and up

### PR DESCRIPTION
**Pull request by ultronjs**
SignIn And Sign Up Pages

### CODE REVIEW

File: src/components/GamesFromBrands.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    { name: "Marvel", linkTo: "/products", imgSrc: MarvelImage },
+    { name: "Ubisoft", linkTo: "/products", imgSrc: UbisoftImage },
+  ];
+  return (
+    <div>
+      <p className="h4 pl-x-small">Games from top brands</p>
+      <div className="container_games_from_top_brands px-small pb-small">
+        {brands &&
+          brands.map((element) => (
+            <Link className="link_wrapper" to={element.linkTo}>
```
Comment (Namrata-J) [inline src/components/GamesFromBrands.jsx:21]: the way you linked to the path is super good!! liked the way you refactored the code here!!

**Review by Namrata-J — APPROVED** [review]

File: src/components/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import "../index.css";
+import React from "react";
+import { Link } from "react-router-dom";
+import { BannerImage } from "../assets";
+
+function Banner() {
+  return (
+    <img className="home_banner" src={BannerImage} alt="" />
```
Comment (fahd23) [inline src/components/Banner.jsx:8]: One Suggesstion, it would be better if you give alternate text for an image

**Review by fahd23 — APPROVED** [review]
Given suggestion, Other than that looks clean

File: src/components/Banner.jsx (hunk 2 above)
Comment (ultronjs) [reply src/components/Banner.jsx:8]: Done

**Comment by ultronjs** [comment]
Merging after 2 approvals

**Pull request merged**